07LessonSeven/homework/index.html

```diff
@@ -5,6 +5,28 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <section>
+    <header>
+      <h1>Profession Portal</h1>
+    </header>
+    <nav>
+      <ul>
+        <li>Home</li>
+        <li>Companies</li>
+        <li>Positions</li>
+      </ul>
+    </nav>
+      <section class="bigbox">
+        <section class="box1">
+          <h1>Hire an employee</h1>
+          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
+          <div>Sign up as company</div>
+        </section>
+        <section class="box2">
+          <h1>Looking for a job</h1>
+          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
+        </section>
+      </section>
+  </section>
 </body>
 </html>
```

Comment:
It looks like you were off to a good start for this assignment. You seem to have all the necessary content (except for the "Sign up as an employee" text) but the finished product is not done. Keep working on it and submit another commit when you are done.